src/routes/Settings.tsx

```diff
@@ -99,12 +98,15 @@ const Settings: React.FC = () => {
       </>
     );
   };
-
   return (
     <div className={classes.container}>
       <Title>Deine Informationen</Title>
       <SettingsCard user={userContext.user} />
-      {!isProjectCoachButNotTutor(userContext.user) && renderSubjects()}
+      {!userContext.user.isTutor &&
+        !(
+          userContext.user.isProjectCoach || userContext.user.isProjectCoachee
```

Comment:
Why do you exclude project coaches here?

Reply:
@lsc0498 if i remember correctly, only tutors get to see the "Deine Fächer" part and only project coaches get to see "Deine Projekte"

Reply:
Yes that's right. But a project coach can also be a tutor and in this case he should see both.

Reply:
will change it as soon as i can then :)